src/analog/CommandBinaryBuffer.ts

```diff
@@ -1800,6 +1809,14 @@ class CommandBinaryBuffer extends BinaryBuffer {
         this.setUint8(parameter.version);
     }
 
+    private getNbiotSslWrite (): IParameterNbiotSslWrite {
```

Comment:
```suggestion
    private getNbiotSslWriteParameter (): IParameterNbiotSslWrite {
```
and in other places like this

Reply:
++